framework/contrib/ice_updater/include/LibcurlUtils.h

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
```

Comment:
Fine by me and if it goes into the framework, yes, the license must change
to LGPL. Totally cool since that would just be MOOSE stuff that we are
using and won't redistribute.
On Sep 22, 2015 3:57 PM, "Derek Gaston" <notifications@github.com> wrote:

> In framework/contrib/ice_updater/include/LibcurlUtils.h
> <https://github.com/idaholab/moose/pull/5717#discussion_r40135676>:
>
> > @@ -0,0 +1,134 @@
> > +/*******************************************************************************
>
> How about the idea of putting all of the ICE agnostic stuff in
> framework/include/utils/url or something? It basically is MOOSE code...
> there's no real reason to keep it separated over in the contrib area. It
> wasn't an external project that we brought in or anything...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5717/files#r40135676>.
>